draft-ietf-httpbis-digest-headers.md

```diff
@@ -1053,7 +1053,7 @@ the whole payload before sending a message (eg. see {{?I-D.thomson-http-mice}}).
 
 Digest may expose information details of encrypted payload when the checksum
 is computed on the unecrypted data.
```

Comment:
typo: unencrypted

Reply:
Looks ok.
Given the labels are being changed to lowercase it might be good to add to the note on the IANA page https://www.iana.org/assignments/http-dig-alg/http-dig-alg.xhtml: "Digest algorithm names are case-insensitive"

Reply:
fixed on master, thanks!

Reply:
@manger imho as algorithms are case-insensitive since RFC3230 you can already add the note to the page: do you need any action on my side for that?